test-cases/test-cases.md

```diff
@@ -0,0 +1,32 @@
+# Test cases
```

Comment:
@m-iwanicki let's add reference to the Cukinia test code

Reply:
Added in [dae787a](https://github.com/TrenchBoot/test-results/pull/1/commits/dae787af8aeb762353cfddc14e22674029c25e41).
Added link with exact commit, after it's merged to master I could change it to always point to newest version on master.